docs/kubeadm.md

```diff
@@ -0,0 +1,210 @@
+---
+---
+
+<style>
+li>.highlighter-rouge {position:relative; top:3px;}
+</style>
+
+## Overview
+
+This quickstart will show you how to easily install a secure Kubernetes cluster on any machines running Linux, using a tool called `kubeadm` which is part of Kubernetes.
+
+This process should work with local VMs, physical servers and/or cloud servers.
+It is intended to be simple enough that you can easily integrate its use into your own automation (Terraform, Chef, Puppet, etc).
+
+**The `kubeadm` tool is currently in alpha but please try it out and give us feedback!**
+
+### tl;dr &mdash; the really short version
+
+You will:
+
+1. Install packages on all machines
+1. Run `kubeadm init` on the machine you want to become a master
+1. Run `kubeadm join` on the machines you want to become nodes
+
+You will then be able to install a pod network with `kubectl apply`.
+
+That's all you will need to bootstrap a secure cluster!
+
+## Prerequisites
+
+1. One or more machines running Ubuntu 16.04 or Fedora 24
+1. 2GB or more of RAM per machine
+1. A network connection between the machines with all ports open
```

Comment:
This is tricky.  By default most folks will run wide open between nodes in the cluster but they may want to lock it down.  Perhaps say "full connectivity" instead of "all ports open".  We can then have an advanced section that talks about how to run over wider links or in a more locked down way?